### PR TITLE
Floatfield

### PR DESCRIPTION
**Pull request by patseng**
@alvinchow86

### CODE REVIEW

File: rohm/fields.py [hunk 1]
```diff
@@ -11,6 +11,8 @@
 from pytz import utc
 
 
+NUMERIC_TYPES = tuple(list(six.integer_types) + [float])  # allow integers and floats
```
Comment (alvinchow86) [inline rohm/fields.py]: can we lowercase this to be consistent with `six`

File: rohm/fields.py [hunk 2]
```diff
@@ -80,10 +82,10 @@ def _from_redis(self, val):
 
 
 class IntegerField(BaseField):
-    allowed_types = six.integer_types
+    allowed_types = NUMERIC_TYPES
```
Comment (alvinchow86) [inline rohm/fields.py]: i'd leave this as is so the client knows there's a type mismatch

File: rohm/fields.py [hunk 3]
```diff
@@ -145,6 +147,16 @@ def _from_redis(self, val):
         return dt
 
 
+class FloatField(BaseField):
+    allowed_types = NUMERIC_TYPES
```
Comment (alvinchow86) [inline rohm/fields.py]: just leave it as [float] for now

Comment (patseng) [inline rohm/fields.py]: I had it as just float, but found that it was yelling at me for trying to serialize ints. I personally dislike that behavior b/c then you have to cast all integers to floats before.

Comment (alvinchow86) [inline rohm/fields.py]: also fix this

File: rohm/fields.py [hunk 4]
```diff
@@ -145,6 +147,16 @@ def _from_redis(self, val):
...
+
+    def _to_redis(self, val):
+        return str(val)
```
Comment (alvinchow86) [inline rohm/fields.py]: lets just make this str(float(val)) then

**Pull request merged**